website/snippets/tutorial-initiate-project.md

```diff
@@ -1,6 +1,6 @@
-Now that you have a repository configured, you can initialize your project and start development in the dbt Cloud IDE:
+Now that you have a repository configured, you can initialize your project and start development in dbt Cloud:
 
-1. Click **Start Developing**.  You can also navigate to the Develop page from ![hamburger menu](/img/hamburger-icon.png) in the top left. It might take a few minutes for your project to spin up for the first time, as it established your git connection, clones your repo, and tests the connection to the warehouse.
+1. Click **Develop** from the top left. It might take a few minutes for your project to spin up for the first time, as it established your git connection, clones your repo, and tests the connection to the warehouse.
```

Comment:
```suggestion
1. Click **Develop** from the upper left. It might take a few minutes for your project to spin up for the first time as it establishes your git connection, clones your repo, and tests the connection to the warehouse.
```